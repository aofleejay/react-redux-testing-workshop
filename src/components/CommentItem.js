import React from 'react'

const CommentItem = (props) => (
  <div className="box">
    <div className="media-content">
      <div className="content">
        <p>
          <strong>{props.title}</strong> <small>{props.author}</small> <small>{props.postat}</small>
          <br />
          {props.body}
        </p>
      </div>
    </div>
  </div>
)

export default CommentItem
