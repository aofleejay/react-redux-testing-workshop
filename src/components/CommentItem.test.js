import React from 'react'
import { shallow } from 'enzyme'
import CommentItem from './CommentItem'

describe('<CommentItem />', () => {
  it('render correctly', () => {
    const props = {
      title: 'title',
      body: 'body',
      author: '@john doe',
      postat: '31m',
    }

    const wrapper = shallow(<CommentItem {...props} />)
    expect(wrapper.html()).toMatchSnapshot()
  })
})