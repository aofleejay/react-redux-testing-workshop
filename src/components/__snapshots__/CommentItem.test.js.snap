// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CommentItem /> render correctly 1`] = `"<div class=\\"box\\"><div class=\\"media-content\\"><div class=\\"content\\"><p><strong>title</strong> <small>@john doe</small> <small>31m</small><br/>body</p></div></div></div>"`;
